BREAKING.md

```diff
@@ -79,6 +81,12 @@ The following legacy events and event parameters have been marked as deprecated
         - This event has been unused and unsupported for some time.
           No replacement API recommended.
 
+
+### Added `fileIsLocked` errorType to DriverErrorType enum
+Added `fileIsLocked` errorType in DriverErrorType enum. This error happens when file is locked for read/write by storage, e.g. whole collection is locked and access is denied, or file is locked for editing.
+
+This is not breaking change yet. But if clients do not add handling for this error, their existing version of applications may start receiving this error in the future, and may not handle it correctly.
```

Comment:
"may not handle it correctly" might be misleading. Maybe add - if application uses dynamic driver loading, current version of application may start receiving these errors from future versions of driver.
I feel like it is also hard to parse. :( So maybe we leave it as is